app/views/users/new.html.haml

```diff
@@ -1,33 +1,36 @@
-.container
-  %h4 Signup
-
-.container
-  .col-md-6.col-md-offset-3
-  = form_for @user do |f|
-    - if @user.errors.any?
-      %h2 Form is invalid
-      %ul
-        - @user.errors.full_messages.each do |message|
-          %li= message
-    - else
-      .field
-        = f.label :name
-        %br/
-        = f.text_field :name
-      .field
-        = f.label :email
-        %br/
-        = f.text_field :email
-      .field
-        = f.label :password
-        %br/
-        = f.password_field :password
-      .field
-        = f.label :password_confirmation
-        %br/
-        = f.password_field :password_confirmation
-        %br/
-        .actions= f.submit "Sign Up"
+.row
+  .col-xs-12.col-sm-8.col-md-4.col-sm-offset-2.col-md-offset-4
+    .panel.panel-default
+      .panel-heading
+        %h1.panel-title
+          Please Sign Up
+      .panel-body
+        %form{:role => "form"}
+          .row
+
+          = form_for @user do |f| 
+            - if @user.errors.any?
+            .alert.alert_danger
+              %p "There was an error in your input, Try again"
+              - for message_error in @user.errors.full_messages
+                %li
+                  * #{message_error}
+            .form-group_tag 
+              - user.errors?([:user][:name]) do
```

Comment:
it is not so much a rabbit hole as it completely halted the entire project, can't sign anyone up. so not sure what happened 
nor can i log in now. 

will continue on but didn't want to get too much into next part until i had this resolved so i knew i had a functional app 

thx 

----- Original Message -----

From: "Levi Cole" <notifications@github.com> 
To: "TeriPastorino/Blocipedia" <Blocipedia@noreply.github.com> 
Cc: "TeriPastorino" <tpasto@comcast.net> 
Sent: Tuesday, July 7, 2015 7:44:36 AM 
Subject: Re: [Blocipedia] Signup confirmation (#1) 



In app/views/users/new.html.haml : 
> +      .panel-heading
> +        %h1.panel-title
> +          Please Sign Up
> +      .panel-body
> +        %form{:role => "form"}
> +          .row
> +
> +          = form_for @user do |f| 
> +            - if @user.errors.any?
> +            .alert.alert_danger
> +              %p "There was an error in your input, Try again"
> +              - for message_error in @user.errors.full_messages
> +                %li
> +                  * #{message_error}
> +            .form-group_tag 
> +              - user.errors?([:user][:name]) do 


You should be able to follow the errors checkpoint here, this is now how we did it in the errors checkpoint. 

I'd not go down this particular rabbit hole if it is keeping you from finishing the project. 

— 
Reply to this email directly or view it on GitHub .